examples/ds3231SqwPin/ds3231SqwPin.ino

```diff
@@ -0,0 +1,48 @@
+#include "RTClib.h"
+
+RTC_DS3231 rtc;
+
+int mode_index = 0;
+
+Ds3231SqwPinMode modes[] = { DS3231_OFF, DS3231_SquareWave1Hz,
+  DS3231_SquareWave1kHz, DS3231_SquareWave4kHz, DS3231_SquareWave8kHz };
+
+void print_mode() {
+  Ds3231SqwPinMode mode = rtc.readSqwPinMode();
+
+  Serial.print("Sqw Pin Mode: ");
+  switch(mode) {
+    case DS3231_OFF:              Serial.println("OFF");       break;
+    case DS3231_SquareWave1Hz:    Serial.println("1Hz");       break;
+    case DS3231_SquareWave1kHz:   Serial.println("1.024kHz");  break;
+    case DS3231_SquareWave4kHz:   Serial.println("4.096kHz");  break;
+    case DS3231_SquareWave8kHz:   Serial.println("8.192kHz");  break;
+    default:                      Serial.println("UNKNOWN");   break;
+  }
+}
+
+void setup () {
+  Serial.begin(57600);
+
+#ifndef ESP8266
+  while (!Serial); // for Leonardo/Micro/Zero
```

Comment:
Trivial, but... There are many micros with USB Serial now, so while I was touching all the examples for #154, I decided to use the language from [the Arduino docs for if (Serial)](https://www.arduino.cc/reference/en/language/functions/communication/serial/ifserial):

`while (!Serial); // wait for serial port to connect. Needed for native USB`

In my opinion it clues new users into the significance of USB serial vs the significance of certain boards.